### ISSUE TITLE
[508] Add page info to Pagination documentation

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#41832]
## Description

To provide optimal screenreader usability, the pagination documentation needs to include a section on adding a page info block above the content. When the page is changed, focus should be moved to this paragraph. Example:

```
Showing 21 through 30 of 50 items, Page 3 of 5

------
card 21
------
card 22
------
...
------
card 30
------

< Prev 1 [2] 3 4 5 Next >

```

<details><summary>Example from Facilities locator</summary>

<img width="1068" alt="Showing 1 through 10 of 17 results for filtered items in the facility location page" src="https://user-images.githubusercontent.com/136959/170119484-e48d67e7-a151-465e-bcaa-6792756e4275.png">
</details>

cc: @joshkimux

## Tasks

- [ ] Add documentation on adding a pager information block that receives focus after changing pages
- [ ] Provide examples
- [ ] Accessibility review of content

## Definition of done

- [ ] All tasks complete

**Comment by caw310** [comment]
Notes from DSC meeting when this was discussed: 
https://docs.google.com/document/d/1ZGwxDt3EQhIzHhEowdhhmbbRRB7a-xtHMr85QrK5E-s/edit?usp=sharing

- Robin to fix bub
- Natalie to sync with Megan.
- Make a component is better choice. 
- Will add it to the queue.
